.gitignore

```diff
@@ -5,3 +5,5 @@ _site
 .git_submodule_init.done.log
 .venv.done.log
 venv
+examples/urgent_evidence/__pycache__
```

Comment:
There's no reason to track `__pycache__` anywhere in this repository, so can this line drop the `examples/urgent_evidence/` part?

Also, can you please restore the end-of-file end-of-line?

Reply:
Updated!  Thank you!